### PR TITLE
[ADD][l10n_ma]: Add new module for Moroccan Localization

### PR DESCRIPTION
**Pull request by KARIZMA-CONSEIL**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:
This is global review for the moroccan localization module since the old module has not been maintained for a long time. The new module:
- Add the account group
- Fix the default proposed taxes
- Fix the default account setting
- Add the ICE identifier for footer reports and invoice reports.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by rgo-odoo** [comment]
@KARIZMA-CONSEIL @saadthaifa  I see you added few `account.group` records, but you never use them, can you please assign them to appropriate accounts. thanks.

File: addons/l10n_ma/data/account_chart_template_data.xml [hunk 1]
```diff
@@ -1,8 +1,3814 @@
...
+        <field name="name">Produits finis en cours de route</field>
+        <field name="code">3156</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id" ref="l10n_kzm_temp_chart"/>
+    </record>
+    <record id="pcg_3158" model="account.account.template">
+        <field name="name">Autres produits finis</field>
+        <field name="code">3158</field>
+        <field name="user_type_id" ref="account.data_account_type_revenue"/>
+        <field name="chart_template_id"
```
Comment (jco-odoo) [inline addons/l10n_ma/data/account_chart_template_data.xml]: acomptes verss?  (something missing?)
Could you tell us why you put this liability account as reconcilable?

**Comment by KARIZMA-CONSEIL** [comment]
Hello @rgo-odoo,
Can you merge the new changes made ?

**Comment by jco-odoo** [comment]
@KARIZMA-CONSEIL  This is quite a heavy change in stable.  If you keep the same tags, ... this might however be possible.  I don't see why you added plenty .po files, just the .pot should suffice.  

@rgo-odoo We can still convert the xml to csv for the account templates as there are quite a lot of them.  (perf)  Can you have a first look also if it is really possible in stable.

**Comment by jco-odoo** [comment]
@KARIZMA-CONSEIL Why is the tax group removed?

**Comment by rgo-odoo** [comment]
@KARIZMA-CONSEIL I have reworked on your code and took important changes and split your PR into two, #32296 will go into V12 and #32297 will go into master, can you please have a look at it. thanks.

**Comment by KARIZMA-CONSEIL** [comment]
@rgo-odoo I have taken a look on the split, it's a good idea.
I see that the commit is still blocked (ci/runbot), can you help me on fixing the error?

**Comment by C3POdoo** [comment]
Dear @KARIZMA-CONSEIL,

Thank you for your contribution but the version 12.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**